js/app/api/slice.js

```diff
@@ -58,11 +61,21 @@ export const apiSlice = createApi({
   // The "endpoints" represent operations and requests for this server
   // nodeId is passed in JSON-LD '@id' key, https://www.w3.org/TR/2014/REC-json-ld-20140116/#node-identifiers
   endpoints: builder => ({
+    subscriptionGenerateOrders: builder.mutation({
+      query: (date) => ({
+        url: 'api/recurrence_rules/generate_orders',
```

Comment:
still looks a hack and a liability to me, I think I would prefer the CRON way

Reply:
The main issue with cron is generating orders on days not covered by the job. For example, if we run it daily, but I want to assign an order for tomorrow which is not created yet. 

As I understood from Paul we might still want orders to be generated automatically without opening the dashboard. So maybe we should combine both approaches, first generate by cron and then still trigger the action onPageLoad